crowdsourcing/viewsets/task.py

```diff
@@ -261,3 +284,37 @@ class CurrencyViewSet(viewsets.ModelViewSet):
 
     queryset = Currency.objects.all()
     serializer_class = CurrencySerializer
+
+
+class ExternalSubmit(APIView):
+    def post(self, request, *args, **kwargs):
+        identifier = request.GET.get('daemo_id', False)
```

Comment:
if we listen on $location change of the iFrame as we discussed and we receive a submission from their page then we can mark it as submitted, workflow doesn't change from the regular POST
workflow is: 1. they submit the form data whichever way they prefer (ajax or regular POST) and 2. they either let us know the user is `done` by location change to something like `/task-done` and we capture that info and continue from there